databases/week2/SchoolDB.sql

```diff
@@ -0,0 +1,26 @@
+CREATE DATABASE SchoolDB;
+
```

Comment:
you might want to add USE statement here to be sure the tables are coming in the right db 😉

Reply:
Yes, you're right :)
In first DB I was thinking that it is not necessary to write it because it already use it, but when I was doing second DB I was thinking like: probably I should have wrote it first DB also :D 

Thank you for your reviews, Alina 🥇